python/lsst/afw/image/wcs.cc

```diff
@@ -55,7 +59,15 @@ PYBIND11_PLUGIN(_wcs) {
     mod.def("makeWcs", (PTR(Wcs) (*)(lsst::afw::coord::Coord const &, lsst::afw::geom::Point2D const &,
             double, double, double, double)) makeWcs);
 
-    py::class_<Wcs, std::shared_ptr<Wcs>, lsst::afw::table::io::Persistable, PersistableFacade<Wcs>> cls(mod, "Wcs");
+    py::class_<Wcs,
+               std::shared_ptr<Wcs>,
+               lsst::daf::base::Citizen,
+               lsst::afw::table::io::Persistable,
+               PersistableFacade<Wcs>> cls(mod, "Wcs");
+
+    py::class_<XYTransformFromWcsPair,
+               std::shared_ptr<XYTransformFromWcsPair>,
+               lsst::afw::geom::XYTransform> clsXYTranFromWcs(mod, "XYTransformFromWcsPair");
```

Comment:
OK, I admit it's way too late to change to following the `pybind11` manual.

But I'm really not willing to use such a crazy long name, either, so I'll add two functions, one per class, and use `cls` internally in each. It may improve readability a bit in any case.